### PR TITLE
Add the linted notebook of RAG with Cohere generate & embedding model…

### PR DESCRIPTION
**Pull request by jameszhyi**
…s from SageMaker JumpStart and LangChain. Update jumpstart-foundation-model README w/ link to Cohere notebook.

*Issue #, if available:*

*Description of changes:*

*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [x] I have tested my notebook(s) and ensured it runs end-to-end
- [x] I have linted my notebook(s) and code using `black-nb -l 100 {path}/{notebook-name}.ipynb`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: introduction_to_amazon_algorithms/jumpstart-foundation-models/question_answering_retrieval_augmented_generation/question_answering_Cohere+langchain_jumpstart.ipynb [hunk 1]
```diff
@@ -0,0 +1,1702 @@
+{
```
Comment (Neo9061) [inline introduction_to_amazon_algorithms/jumpstart-foundation-models/question_answering_retrieval_augmented_generation/question_answering_Cohere+langchain_jumpstart.ipynb:1]: <p>please remove the output of each cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4409/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (Neo9061) [inline introduction_to_amazon_algorithms/jumpstart-foundation-models/question_answering_retrieval_augmented_generation/question_answering_Cohere+langchain_jumpstart.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    Cohere_Generate_Endpoint_Name = "Endpoint-Cohere-Generate-Model-Command-Light-01"
</pre><p>could you please use prefix of jumpstart-example-raglc- as the endpoint name?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4409/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (Neo9061) [inline introduction_to_amazon_algorithms/jumpstart-foundation-models/question_answering_retrieval_augmented_generation/question_answering_Cohere+langchain_jumpstart.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    cohere_embedding_package = "cohere-multilingual-small-v1-1-e0210aae7f8135adaec4b199df604c4c"
</pre><p>Do we have other variants of emebdding models? I saw this is small size. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4409/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (Neo9061) [inline introduction_to_amazon_algorithms/jumpstart-foundation-models/question_answering_retrieval_augmented_generation/question_answering_Cohere+langchain_jumpstart.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    Cohere_Embedding_Endpoint_Name = "Endpoint-Cohere-Multilingual-Embedding-Model-01"
</pre><p>same comment here:  could you please use prefix of jumpstart-example-raglc- as the endpoint name? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4409/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (jameszhyi) [reply introduction_to_amazon_algorithms/jumpstart-foundation-models/question_answering_retrieval_augmented_generation/question_answering_Cohere+langchain_jumpstart.ipynb:1]: The outputs from each cell were removed in the latest version.

Comment (jameszhyi) [reply introduction_to_amazon_algorithms/jumpstart-foundation-models/question_answering_retrieval_augmented_generation/question_answering_Cohere+langchain_jumpstart.ipynb:1]: The prefix was added in the endpoint name.

Comment (jameszhyi) [reply introduction_to_amazon_algorithms/jumpstart-foundation-models/question_answering_retrieval_augmented_generation/question_answering_Cohere+langchain_jumpstart.ipynb:1]: The prefix was added in the endpoint name.

File: introduction_to_amazon_algorithms/jumpstart-foundation-models/question_answering_retrieval_augmented_generation/question_answering_Cohere+langchain_jumpstart.ipynb [hunk 2]
```diff
@@ -0,0 +1,1461 @@
...
```
Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart-foundation-models/question_answering_retrieval_augmented_generation/question_answering_Cohere+langchain_jumpstart.ipynb:1]: <p>Add badges as described in CONTRIBUTiNG.md </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4409/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (jameszhyi) [reply introduction_to_amazon_algorithms/jumpstart-foundation-models/question_answering_retrieval_augmented_generation/question_answering_Cohere+langchain_jumpstart.ipynb:1]: Thanks! The badges were added as described in CONTRIBUTiNG.md.

**Pull request merged**